src/drivers/boards/auav-x21/board_config.h

```diff
@@ -210,6 +210,9 @@
 
 #define BOARD_NAME "AUAV_X21"
 
+// Run the FMU task on its own thread to reduce jitter
+#define PX4FMU_TASK
```

Comment:
It is not a nit. see https://github.com/PX4/Firmware/issues/1619

Please use the form 

```
#define PX4FMU_TASK 1
#if defined(PX4FMU_TASK)
```

If you want to go one step further, and allow the board to control the task priority use
```
#define PX4FMU_TASK_PRIORITY xxx
#if defined(PX4FMU_TASK_PRIORITY )
```